lib/ruy/conditions/day_of_week.rb

```diff
@@ -4,34 +4,40 @@
 module Ruy
   module Conditions
 
-    # Expects that a Time object's date corresponds to a specified day of the week.
+    # Expects that a Time object's date corresponds to a specified day of
+    # the week
+    #
     class DayOfWeek < Condition
       DAYS_INTO_WEEK = %w(sunday monday tuesday wednesday thursday friday saturday)
 
-      attr_reader :day, :attr, :tz_identifier
+      attr_reader :day, :key, :tz_identifier
 
       # @param day
-      # @param attr
+      # @param key
       # @param tz_identifier
-      def initialize(day, attr, tz_identifier = 'UTC')
+      # @example to check that the day in 'time' is wednesday
+      #   DayOfWeek.new(:wednesday, :time)
+      # @example to check that the day in 'time' is wednesday in CST
+      #   DayOfWeek.new(:wednesday, :time, 'CST')
+      def initialize(day, key, tz_identifier = 'UTC')
         super
         @day = day
-        @attr = attr
+        @key = key
         @tz_identifier = tz_identifier
       end
 
       def ==(o)
         o.kind_of?(DayOfWeek) &&
           o.day == @day &&
-          o.attr == @attr &&
+          o.key == @key &&
           o.tz_identifier == @tz_identifier
       end
 
       protected
 
-      def evaluate(value)
+      def evaluate(date)
```

Comment:
`evaluate` is the most important method of a `Condition` class, and it basically takes charge of performing the evaluation the class is meant to do, so the description of what it does lies in the description of the class. What I can do is to add a `@see` directive to reference `super`, but basically the documentation would be the documentation of the class again.